### ISSUE TITLE
';' is not parsed as the beginning of a comment

### ISSUE THREAD

**Issue by dod38fr** [dod38fr/config-model-systemd#1]
Systemd configuration file allow comments to begin with `#` or `;`. The latter character is not taken into account. 

symtom: while reading lirc service file on Debian: `cme dump systemd` produces:
```
Warning: ; User is confusingly close to User (edit distance is 2). Is there a typo ?
```
Note the `; ` before `User`.

**Comment by dod38fr** [comment]
Depends on https://github.com/dod38fr/config-model/issues/10

**Comment by dod38fr** [comment]
Fixed in 0.232.1
